remoting/resources/web/rdf4j.xml

```diff
@@ -24,6 +24,10 @@
 
 <web-app xmlns="http://java.sun.com/xml/ns/j2ee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/j2ee http://java.sun.com/xml/ns/j2ee/web-app_2_4.xsd" version="2.4">
     <display-name>RDF4J Server</display-name>
+    <!-- absolute ordering to prevent IllegalArgumentExcepition on startup -->
```

Comment:
That XML warning never occured to me. Seems to be an eclipse IDE issue. But does not harm